docs/source/compression/compression_evaluator.rst

```diff
@@ -0,0 +1,311 @@
+.. _compression-evaluator:
```

Comment:
No need for this. This is equivalent to :doc:`compression_evaluator`.

Reply:
I tried, but this is more convenient, because I refer this doc in a shared docstring ([nni/algorithms/compression/v2/pytorch/utils/docstring.py](https://github.com/microsoft/nni/pull/5016/files#diff-9c593e0ae1a50892568ee7f2dccdd64eef725abae812656d0ce53a2425cb8b3e)), and I don't know the relative position

Reply:
You can use absolute path, i.e., `/compression/compression_evaluator`.

Reply:
it works, fixed